src/test/groovy/com/mechanitis/demo/spock/OrderingServiceTest.groovy

```diff
@@ -0,0 +1,100 @@
+package com.mechanitis.demo.spock
+
+import com.synacy.gradprogram.spock.exercise.*
+import spock.lang.Specification
+
+class OrderingServiceTest extends Specification {
+
+    OrderingService service
+    Cart cart
+
+    def items = [
+            new Item("Chicken Cordon", 25, ItemType.FOOD),
+            new Item("Trouser", 1500, ItemType.CLOTHING),
+            new Item("Smart-TV", 3500, ItemType.APPLIANCE),
+            new Item("IPhone14Pro", 62000, ItemType.GADGET),
+            new Item("Pancit Malabon", 110, ItemType.FOOD)
+    ]
+
+    def setup() {
+        service = new OrderingService()
+    }
+
+    def "Validate the Items if there have food Order on the Cart"() {
```

Comment:
Let's try to follow the naming convention of including the method name here:
```
def "methodName should ..."()
```

Same with the other tests.